packages/grid/helm/syft/values.yaml

```
@@ -201,6 +203,7 @@ node:
 
 ingress:
   hostname: null # do not make this localhost
+  annotations: {}
```

```suggestion
  annotations: null
```
NIT: we could have it either as null or empty collection ({})
both evaluate to False in helm
we could keep it as  null to maintain uniformity with other values in  values.yaml